domain/secretbackend/state/types.go

```diff
@@ -23,12 +25,19 @@ type ModelSecretBackend struct {
 	// Name is the name of the model.
 	Name string `db:"name"`
 	// Type is the type of the model.
-	Type coremodel.ModelType `db:"type"`
+	Type coremodel.ModelType `db:"model_type_type"`
```

Comment:
Yuk. I realise it comes from here
```
...
mt.type       AS model_type_type,
```
Why can't we just do `mt.type       AS model_type,`

Reply:
Good pick up. Changing now.